### PR TITLE
Fix code samples and add CI test framwork

### PR DESCRIPTION
**Pull request by tom-rosewell**
Testing the code sample test framework - WIP

### CODE REVIEW

**Comment by tom-rosewell** [comment]
Finished, code samples updated, ready for implementation

File: tests/node_modules/bats-assert/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Change Log
```
Comment (RichardSmedley) [inline tests/node_modules/bats-assert/CHANGELOG.md]: Do all of these non-functional files need to be in here?
(Licence, readme, etc)

Comment (tom-rosewell) [reply tests/node_modules/bats-assert/CHANGELOG.md]: Removed

**Review by RichardSmedley — APPROVED** [review]
Thanks @tom-rosewell   :+1:

**Pull request merged**